package.json

```diff
@@ -1,6 +1,14 @@
 {
   "name": "@fractal-framework/fractal-contracts",
-  "version": "0.1.0",
+  "version": "1.0.0",
+  "files": [
+    "dist",
```

Comment:
What is the purpose of "dist" here?

Reply:
Re-using approved code from previous fractal-contracts-repo that Adam already approved.